addOns/exim/src/main/resources/org/zaproxy/addon/exim/resources/Messages.properties

```diff
@@ -42,4 +54,9 @@ exim.savexml.popup.option = Save XML
 
 exim.har.file.description = HTTP Archive File (*.har)
 exim.har.file.save.error = Error saving file to {0}.
-exim.har.popup.option = Save Selected Entries as HAR (HTTP Archive File)
\ No newline at end of file
+exim.har.popup.option = Save Selected Entries as HAR (HTTP Archive File)
+
+exim.options.value.type.har = har
```

Comment:
It would be better if these matched the names used in other UI dialogues (e.g. `HTTP Archive File`), same for other strings.